index.html

```diff
@@ -15,5 +15,107 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+
+<header>
+    <div id="header">
+        <div class="logo">
+            <img src="favicon.ico" alt="logo Karma"/>
+        </div>
+
+        <nav>
+            <div id="nav-bar">
+                <button class="nav-item"><strong>Meet Karma</strong></button>
+                <button class="nav-item">How it works</button>
+                <a class="nav-item" href="store.html">Store</a>
+                <button class="nav-item">Blog</button>
+                <button class="nav-item">Help</button>
+                <button class="nav-item">Login</button>
+
+            </div>
+
+        </nav>
+
+    </div>
+
+
+<div class="container">
+    <img class=" image" src="img/first-background.jpg" alt="person sitting on a table"/>
+    <div class="content">
+        <h1 class="heading">Introducing Karma</h1>
+        <h2 class="sub_heading">Bring Wifi with you, everywhere you go.</h2>
+        <button class="btn-primary">Learn More</button>
+    </div>
+
+</div>
+
+</header>
+<main>
+    <h1 id="karma_heading">Everyone need a little Karma</h1>
+    <div id="info-icons">
+        <div>
+            <img class="icon" src="img/icon-devices.svg" alt="logo"/>
+            <p class="description">Internet for all devices</p>
+        </div>
+        <div>
+            <img class="icon" src="img/icon-coffee.svg" alt="image of coffee"/>
+            <p class="description">Boost your productivity</p>
+        </div>
+        <div>
+            <img class="icon" src="img/icon-refill.svg" alt="image of refill"/>
+            <p class="description">Pay as you go</p>
+        </div>
+
+    </div>
+
+</main>
+
+<div class="container2">
+    <img src="level-2/homepage-feature.png" alt="a woman reading" />
+    <div class="content2">
+        <p class="info_connection">"Wherever I am, I just don't worry about my connection anymore!"</p>
+       <a href="store.html">
+        <button class="btn-secondary">Get Karma Today</button>
+        </a>
+
+    </div>
+</div>
+
+<hr>
+
+
+<footer>
+    <p id="paragraph_join">Join us on</p><br>
+
+    <div id="contacts">
+        <div>
+        <button class="container-social">
+            <img class="img_social" src="img/twitter-icon.svg" alt="twitter symbol"/> 
+        </button>
+        </div>
+
+        <div>
+            <button class="container-social">
+                <img class="img_social" src="img/facebook-icon.svg" alt="facebook symbol"/>
+
+            </button>
+        </div>
+
+        <div>
+            <button class="container-social">
+                <img class="img_social" src="img/instagram-icon.svg" alt="instagram symbol"/>
+            </button>
+
+        </div>
```

Comment:
The code is not indented and formatted consistently, which makes it harder to read the code.
You can install a VS code extension called *Prettier* which automatically formats your code.
You can read about it here https://syllabus.codeyourfuture.io/guides/code-style-guide